src/pj_init.c

```diff
@@ -685,6 +680,12 @@ pj_init_ctx(projCtx ctx, int argc, char **argv) {
             }
         PIN = 0;
     }
+    else {
```

Comment:
Yes - this seems to repair a genuine blunder (although it would be nice if geod_init included a check for null pointers, but I think, given that the geod-code seems to be autotranslated from the C++-code of GeographicLib, would be too much to ask for, so this solution is definitely preferrable)